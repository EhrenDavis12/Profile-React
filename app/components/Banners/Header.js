import PropTypes from 'prop-types';
import React from 'react';
import styled from 'styled-components';

export const H2 = styled.h2`
  border-style: solid;
  border-width: 0px;
  border-bottom-width: 5px;
  margin-left: 20px;
`;

export const Row12 = styled.div`
  margin: 5px;
  class: row col-lg-12;
`;

function Header(props) {
  return (
    <Row12>
      <H2>{props.children}</H2>
    </Row12>
  );
}

Header.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Header;
