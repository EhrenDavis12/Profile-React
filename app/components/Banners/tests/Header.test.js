import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Header from '../Header';

describe('<Header />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<Header>Hello Wold</Header>);
    expect(renderedComponent).toMatchSnapshot();
  });
});
