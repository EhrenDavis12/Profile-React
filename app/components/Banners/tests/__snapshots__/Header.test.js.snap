// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render the page 1`] = `
<Header__Row12>
  <Header__H2>
    Hello Wold
  </Header__H2>
</Header__Row12>
`;
